jgiven-gradle-plugin/gradle.properties

```diff
@@ -0,0 +1 @@
+version=1.0.0
```

Comment:
is this file even necessary? 
The gradle plugin projects might have access to the top-level properties file.

Reply:
if not, it is probably better to create a version variable in build.gradle and pass around that one

Reply:
It works without it, I removed it.